Tests are set up to use the installed library and headers.
Since we test before installing, this points to the files in the build dir instead.
author: micketeer@gmail.com
diff -ru api_tests.orig/Makefile api_tests/Makefile
--- api_tests.orig/Makefile	2026-10-18 02:19:41.807634085 +0200
+++ api_tests/Makefile	2026-10-18 02:20:14.359126867 +0200
@@ -9,10 +9,10 @@
 test_bins: $(SKBB_TESTS)
 
 test_distance.exe: test_distance.c
-	$(CC) $(CFLAGS) -std=c99 -O0 -g $< $(LDFLAGS) -lskbb -o $@
+	$(CC) $(CFLAGS) -std=c99 -O0 -g $< $(LDFLAGS) ../src/libskbb.* -o $@
 
 test_ordination.exe: test_ordination.c
-	$(CC) $(CFLAGS) -std=c99 -O0 -g $< $(LDFLAGS) -lskbb -o $@
+	$(CC) $(CFLAGS) -std=c99 -O0 -g $< $(LDFLAGS) ../src/libskbb.* -o $@
 
 clean:
 	-rm -f *.o *.exe
diff -ru api_tests.orig/test_distance.c api_tests/test_distance.c
--- api_tests.orig/test_distance.c	2026-10-18 02:19:41.807634085 +0200
+++ api_tests/test_distance.c	2026-10-18 02:20:53.460517590 +0200
@@ -7,8 +7,8 @@
  * See LICENSE file for more details
  */
 
-#include "scikit-bio-binaries/distance.h"
-#include "scikit-bio-binaries/util.h"
+#include "../src/extern/distance.h"
+#include "../src/extern/util.h"
 #include <unistd.h>
 #include <stdio.h>
 #include <math.h>
diff -ru api_tests.orig/test_ordination.c api_tests/test_ordination.c
--- api_tests.orig/test_ordination.c	2026-10-18 02:19:41.808634070 +0200
+++ api_tests/test_ordination.c	2026-10-18 02:21:15.812169308 +0200
@@ -7,8 +7,8 @@
  * See LICENSE file for more details
  */
 
-#include "scikit-bio-binaries/ordination.h"
-#include "scikit-bio-binaries/util.h"
+#include "../src/extern/ordination.h"
+#include "../src/extern/util.h"
 #include <unistd.h>
 #include <stdio.h>
 #include <stdlib.h>
